ui/lib/core/addon/components/search-select.js

```diff
@@ -1,99 +1,123 @@
  * @example
- * <SearchSelect @id="group-policies" @models={{["policies/acl"]}} @onChange={{onChange}} @selectLimit={{2}} @inputValue={{get model valuePath}} @helpText="Policies associated with this group" @label="Policies" @fallbackComponent="string-list" />
- *
- * @param {string} id - The name of the form field
- * @param {Array} models - An array of model types to fetch from the API.
- * @param {object} [queryObject] - object passed as query options to this.store.query(). NOTE: will override this.backend
+ *  <SearchSelect
+ *    id="policy"
+ *    models={{array "policies/acl"}}
+ *    onChange={{this.onChange}}
+ *    inputValue={{get @model this.valuePath}}
+ *    wildcardLabel="role"
+ *    fallbackComponent="string-list"
+ *    selectLimit={{1}}
+ *    backend={{@model.backend}}
+ *    disallowNewItems={{true}}
+ *    class={{if this.validationError "dropdown-has-error-border"}}
+ * />
+ * 
+ // * component functionality
  * @param {function} onChange - The onchange action for this form field. ** SEE UTIL ** search-select-has-many.js if selecting models from a hasMany relationship
- * @param {string | Array} inputValue -  A comma-separated string or an array of strings -- array of ids for models.
- * @param {string} label - Label for this form field
- * @param {string} fallbackComponent - name of component to be rendered if the API call 403s
- * @param {string} [backend] - name of the backend if the query for options needs additional information (eg. secret backend)
+ * @param {array} [inputValue] - Array of strings corresponding to the input's initial value, e.g. an array of model ids that on edit will appear as selected items below the input
  * @param {boolean} [disallowNewItems=false] - Controls whether or not the user can add a new item if none found
- * @param {boolean} [passObject=false] - When true, the onChange callback returns an array of objects with id (string) and isNew (boolean) (instead of an array of id strings)
- * @param {array} [objectKeys=null] - Array of values that correlate to model attrs. When passObject=true, objectKeys are added to the passed object. NOTE: make 'id' as the first element in objectKeys if you do not want to override the default of 'id'
- * @param {string} [helpText] - Text to be displayed in the info tooltip for this form field
- * @param {number} [selectLimit] - A number that sets the limit to how many select options they can choose
+ * @param {boolean} [shouldRenderName=false] - By default an item's id renders in the dropdown, `true` displays the name with its id in smaller text beside it *NOTE: the boolean flips automatically with 'identity' models or if this.idKey !== 'id'
+ * @param {array} [parentHandleSelection] - Array of selected items if the parent is keeping track of selections, see mfa-login-enforcement-form.js
```

Comment:
It looks like the arg name should be `parentManageSelected`

Reply:
Thank you for catching! I hadn't updated the documentation